gloo/docs/gloo_routing/virtual_services/authentication/external_auth/_index.md

```diff
@@ -7,4 +7,11 @@ description: Authenticating Virtual Services with Gloo's built-in Auth server, w
 Enterprise Gloo comes with a built-in authentication server, and several different forms of authentication are 
 supported out of the box, which can be configured per **VirtualService**. 
 
-{{% children description="true" %}}
\ No newline at end of file
+The external-auth service itself can be configured via helm:
+
+| option                                                    | type     | description                                                                                                                                                                                                                                                    |
+| --------------------------------------------------------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| global.extensions.extAuth.envoySidecar                    | bool     | deploy ext-auth in the gateway-proxy pod, as a sidecar to envoy. communicates over unix domain socket instead of TCP. default is `false` |
```

Comment:
Nit: 

```suggestion
| global.extensions.extAuth.envoySidecar                    | bool     | Deploy ext-auth in the gateway-proxy pod, as a sidecar to envoy. Communicates over unix domain socket instead of TCP. Default is `false` |
```